engine/src/main/java/org/terasology/rendering/nui/internal/NUIManagerInternal.java

```diff
@@ -145,7 +145,10 @@ public NUIManagerInternal(TerasologyCanvasRenderer renderer, Context context) {
         TabbingManager.setFocusManager(this);
 
         // NOTE: Fix for tests
-        if (bindsManager != null) {
+        if (bindsManager != null
+                && bindsManager.getBindsConfig().hasBinds(new SimpleUri("engine:tabbingUI"))
+                && bindsManager.getBindsConfig().hasBinds(new SimpleUri("engine:tabbingModifier"))
+                && bindsManager.getBindsConfig().hasBinds(new SimpleUri("engine:activate"))) {
```

Comment:
I'm worried about this one. If we skip over setting up this up here, when _will_ it get done?

I'd want to either fix whatever is supposed to set those bindings so it reliably runs before we get here, or move this to a callback or event handler that is called once the bindsManager is all configured.